cstrml/market/src/mock.rs

```diff
@@ -148,7 +148,7 @@ impl Trait for Test {
     type CurrencyToBalance = CurrencyToVoteHandler;
     type Event = ();
     type Randomness = TestRandomness;
-    type OrderInspector = TestOrderInspector;
+    type OrderInspector = Swork;
```

Comment:
This is useless. I would remove `market/*` and rename `market_v2 => market` in the next PR

Reply:
Sure